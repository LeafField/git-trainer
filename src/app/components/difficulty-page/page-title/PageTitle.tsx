"use client";
import React, { FC } from "react";

type Props = {
  title: string;
};

const PageTitle: FC<Props> = ({ title }) => {
  return (
    <h1 className="text-3xl lg:text-4xl pt-[3.62rem] text-white row-start-1 lg:col-start-2 lg:col-end-4 self-center ">
      {title}
    </h1>
  );
};

export default PageTitle;
