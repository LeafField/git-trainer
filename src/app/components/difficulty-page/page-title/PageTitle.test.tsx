import { render, screen } from "@testing-library/react";
import PageTitle from "./PageTitle";

describe("PageTitle", () => {
  it("初級編:GitHub Flowが表示される", () => {
    render(<PageTitle title="初級編:GitHub Flow" />);
    expect(screen.getByText("初級編:GitHub Flow")).toBeInTheDocument();
  });

  it("titleの中身を変更した時も正しく表示される", () => {
    render(<PageTitle title="中級編:GitHub Flow" />);
    expect(screen.getByText("中級編:GitHub Flow")).toBeInTheDocument();
  });
});
