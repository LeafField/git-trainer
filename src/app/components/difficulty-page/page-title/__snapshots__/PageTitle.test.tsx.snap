// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle スナップショットテスト 1`] = `
<div>
  <h1
    class="text-3xl lg:text-4xl pt-[3.62rem] text-white row-start-1 lg:col-start-2 lg:col-end-4 self-center "
  >
    初級編:GitHub Flow
  </h1>
</div>
`;
